conf/nd96isr_h100_v5.conf

```diff
@@ -35,11 +35,14 @@
  * || check_hw_eth ib6
  * || check_hw_eth ib7
  * || check_hw_eth docker0
+# * || check_vbios_version 96.00.74.00.01
```

Comment:
can we remove the commented vbios check?

Reply:
Removed